app/controllers/admin/users_controller.rb

```diff
@@ -16,7 +19,10 @@ def index
     respond_to do |format|
       format.html { @users = [] }
       format.json { render "index" }
-    end
+      format.xml { render xml:  @users }
```

Comment:
Are we using XML? If not can we remove it?

Reply:
Removed it

Reply:
thanks, looks good